src/main/kotlin/no/nav/syfo/consumer/behandlendeenhet/BehandlendeEnhetConsumer.kt

```diff
@@ -18,7 +19,7 @@ class BehandlendeEnhetConsumer(
     @Value("\${syfobehandlendeenhet.url}") private val baseUrl: String,
     @Value("\${syfobehandlendeenhet.client.id}") private val syfobehandlendeenhetClientId: String,
     private val metric: Metric,
-    private val template: RestTemplate
+    @Qualifier("restTemplateWithProxy") private val template: RestTemplate
```

Comment:
I følge nais-doc'en skal ikke dette være nødvendig, men det gjør kanskje ikke noen skade heller: 
"The application on-premises should not have to use webproxy to reach these ingresses."

Reply:
Jeg kan teste det det uten.

Reply:
Det fungerte kun med restTemplateWithProxy. Uten det så fikk jeg 404 slik koden er nå.